tracing-core/src/field.rs

```diff
@@ -169,8 +169,15 @@ pub struct Iter {
 /// `examples/counters.rs`, which demonstrates a very simple metrics system
 /// implemented using `tracing`.
 ///
-/// **Note:** the `record_error` trait method is only available when the Rust
-/// standard library is present, as it requires the `std::error::Error` trait.
+/// <div class="information">
+///     <div class="tooltip ignore" style="">ⓘ<span class="tooltiptext">Note</span></div>
+/// </div>
+/// <div class="example-wrap" style="display:inline-block">
+/// <pre class="ignore" style="white-space:normal;font:inherit;">
+/// <strong>Note</strong>: The <code>record_error</code> trait method is only
+/// available when the Rust standard library is present, as it requires the `
```

Comment:
extra backtick
```suggestion
/// available when the Rust standard library is present, as it requires the 
```

![Screenshot from 2020-06-29 12-00-31](https://user-images.githubusercontent.com/1993852/86045242-29aa1480-ba00-11ea-8e97-f05606c95f58.png)